### PR TITLE
tree

### PR DESCRIPTION
**Pull request by Damian96gz**

### CODE REVIEW

File: 01-python/infrastructure/tree.py [hunk 1]
```diff
@@ -1,3 +1,26 @@
 def draw_tree():
-    print("tree not found")
+    #print("tree not found")
```
Comment (gallettilance2) [inline 01-python/infrastructure/tree.py]: please remove this comment

Comment (Damian96gz) [reply 01-python/infrastructure/tree.py]: Removed! ^ ^

**Comment by Damian96gz** [comment]
added variable

File: 01-python/infrastructure/tree.py [hunk 2]
```diff
@@ -29,3 +63,4 @@ def draw_tree(height = 5):
             print("o")
 
         return
+>>>>>>> 0e9345cda92eb5709818714a905c011dfd8d30ce
```
Comment (gallettilance) [inline 01-python/infrastructure/tree.py]: this is a git conflict. Please remove this code.

File: 01-python/infrastructure/tree.py [hunk 3]
```diff
@@ -1,3 +1,37 @@
+<<<<<<< HEAD
+def draw_tree(height=5):
```
Comment (gallettilance) [inline 01-python/infrastructure/tree.py]: what if height is <0?

File: 01-python/infrastructure/tree.py [hunk 4]
```diff
@@ -1,3 +1,37 @@
+<<<<<<< HEAD
```
Comment (gallettilance) [inline 01-python/infrastructure/tree.py]: remove this line

**Pull request closed without merging**